src/Lumi/Components2/Button.purs

```diff
@@ -85,22 +83,16 @@ button =
       else
         props.content
 
-primary :: PropsModifier ButtonProps
-primary = identity
-
-secondary :: PropsModifier ButtonProps
-secondary =
+_secondary :: forall props. PropsModifier ( kind :: ButtonKind | props )
```

Comment:
Is the idea that PropsModifier values should always have a leading underscore?

Reply:
I started using it to differentiate the primary styles from the modifiers, like `box` and `_row` in the styles module and it seemed to make sense here to. It's mainly to indicate that it isn't super useful when it isn't used with the base styles and keep it easy to find the primary style import. Conventions over types could be a red flag though..